tapglue-android-sdk/src/main/java/com/tapglue/networking/TGNetworkManager.java

```diff
@@ -415,8 +424,40 @@ private void performRequest(@NonNull final TGRequest request, boolean flushIfPos
                     sendErrorToCallbacks(request.getCallback(), TGRequestErrorType.ErrorType.UNSUPPORTED_INPUT);
                 }
                 else {
```

Comment:
If I understand CR (you want to switch from "if(){code1} else{code2}" to "if (){code1 break;} code2; break; - This is just an another convention - it changes nothing in application flow. Most of app is written using  if {} else{} instead of breaking from switch at this place. This is possible, but would require more rework - and as both conventions are acceptable I suggest leaving it as it is.